test/unit/dashboard/launch.test.js

```diff
@@ -0,0 +1,69 @@
+const expect = require('chai').expect;
+const io = require('socket.io-client');
+const { nanoid } = require('nanoid');
+
+const id = nanoid(16);
+
+// unit under test
+const Server = require('../../../dashboard/server');
+
+describe('Launch broker', () => {
+    let launchedServer;
+
+    before(() => {
+        launchedServer = Server();
+    });
+
+    after(() => {
+        launchedServer.io.close();
+        launchedServer.server.close();
+    });
+
+    describe('Run event emits', () => {
```

Comment:
All nested `describe` blocks and `it` blocks combined should for a sentence. If it does not, maybe we need to restructure the tests to make them behaviour driven.